front/app/components/admin/PostManager/components/PostTable/Row/IdeaRow.tsx

```diff
@@ -208,26 +205,12 @@ const IdeaRow = ({
           const newTopics = uniq(currentTopics?.concat(dropResult.id));
 
           ideaIds.forEach((ideaId) => {
-            updateIdea(
-              {
-                id: ideaId,
-                requestBody: {
-                  topic_ids: newTopics,
-                },
+            updateIdea({
+              id: ideaId,
+              requestBody: {
+                topic_ids: newTopics,
               },
-              {
-                onSuccess: () => {
```

Comment:
@luucvanderzee Any idea why we needed to refetch the allowed project topics and the list of topics when updating an idea? Doesn't make much sense to me but maybe there was a reason?

Reply:
The allowed project topics might make sense: if you initially only specified topics A and B as allowed input topics, but then as an admin you start tagging ideas with topic C in the input manager, maybe the BE is smart enough to automatically update the allowed input topics to include C? We would have to ask tho, I don't remember how/if this was implemented

As for the general topics, I'm not sure

Reply:
This drag and drop UI doesn't work very well BTW- but that's also the case on master

Reply:
Alright! thanks for checking it out 🙌